Packages/com.unity.inputsystem/InputSystem/Plugins/XR/MagicLeap.cs

```diff
@@ -0,0 +1,121 @@
+using UnityEngine.InputSystem.Controls;
+using UnityEngine.InputSystem.Layouts;
+
+namespace UnityEngine.InputSystem.XR
+{
+    [InputControlLayout]
+    public class MagicLeapLightwear : XRHMD
+    {
+        public IntegerControl trackingState { get; private set; }
+        public ButtonControl isTracked { get; private set; }
+        public Vector3Control devicePosition { get; private set; }
+        public QuaternionControl deviceRotation { get; private set; }
+        public Vector3Control centerEyePosition { get; private set; }
+        public QuaternionControl centerEyeRotation { get; private set; }
+        public AxisControl confidence { get; private set; }
+        public AxisControl fixationPointConfidence { get; private set; }
+        public AxisControl eyeLeftCenterConfidence { get; private set; }
+        public AxisControl eyeRightCenterConfidence { get; private set; }
```

Comment:
Do these need `[InputControl]` attributes now? Seeing them on the other layouts but not here. Probably something I'm missing.

Reply:
They will, adding them separately